docs/src/Theory/Atmos/Microphysics.md

```diff
@@ -714,6 +720,115 @@ where:
+  [accretion](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/Microphysics/#Accretion),
+  [rain evaporation](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/Microphysics/#Rain-evaporation-and-snow-sublimation), and
+  [snow melt](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/Microphysics/#Snow-melt)
+  processes.
+Similarily, the source of snow ``\mathcal{S}_{q_{sno}}`` is a sum of the
+  [autoconversion](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/Microphysics/#Snow-autoconversion),
+  [accretion](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/Microphysics/#Accretion),
+  [snow deposition/sublimation](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/Microphysics/#Rain-evaporation-and-snow-sublimation), and
+  [snow melt](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/Microphysics/#Snow-melt)
+  processes.
+The sink of total water is equal to the sum of the rain and snow sources:
+  ``\mathcal{S}_{q_{tot}}`` = -``\mathcal{S}_{q_{rai}}`` - ``\mathcal{S}_{q_{sno}}``.
+The sink of cloud liquid water ``\mathcal{S}_{q_{liq}}`` is the sum of
+  rain autoconversion,
+  rain accretion with cloud liquid water, and
+  snow accretion with cloud liquid water processes.
+The sink of cloud ice ``\mathcal{S}_{q_{ice}}`` is the sum of
+  snow autoconversion,
+  snow accretion with cloud ice, and
+  rain accretion with cloud ice processes.
+Following the conservation equations for
+[mass](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/AtmosModel/#Mass),
+[moisture](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/AtmosModel/#Moisture), and
+[precipitation](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/AtmosModel/#Precipitating-Species)
+the ``\mathcal{S}_{q_{tot}}`` sink has to be multiplied by ``\rho`` before
+  adding it as one of the sink terms to both ``\rho`` and ``\rho q_{tot}``
+  state variables.
+The ``\mathcal{S}_{q_{liq}}`` and ``\mathcal{S}_{q_{ice}}`` sinks have to be multiplied by ``\rho``
+  before adding them as one of the sink terms to ``\rho q_{liq}``  and ``\rho q_{ice}`` state variables.
+The ``\mathcal{S}_{q_{rai}}`` and ``\mathcal{S}_{q_{sno}}`` sources have to be multiplied by ``\rho``
+  before adding them as one of the source terms to ``\rho q_{rai}`` and ``\rho q_{sno}``state variables.
+For the conservation equation for
+[total energy](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/AtmosModel/#Energy)
+  the contribution may come from removing cloud condensate outside of the working fluid
+  or changing phase and releasing latent heat outside of the working fluid.
+The contribution from
+  cloud liquid water to rain autoconversion,
+  cloud liquid water accretion by rain,
+  and rain evaporation
+  is computed as
+```math
+\begin{equation}
+\left. \mathcal{S}_{\rho e}} \right|_{precip} =
+  - (I_l + \Phi) \rho \, \mathcal{S}_{q_{rai}}
+\end{equation}
```

Comment:
This equation is broken also

Reply:
I hope I fixed all of them. Lets see in the next preview